_build_dependencies.sh

```diff
@@ -7,5 +7,3 @@ export NODE_VERSION=12
 export BYOND_MAJOR=513
 # Byond Minor
 export BYOND_MINOR=1528
-# For the RUSTG library. Not actually installed by CI but kept as a reference
-export RUSTG_VERSION=2.1-P
```

Comment:
No longer required, and hasnt been for a while.

Reply:
doesn't this PR literally use rustg for its parsing???

Reply:
@ZeWaka I meant in the CI context. We package `librustg.so` with the repo, and this script was horrifically out of date (We are at `4.5-P3` as of now. We have version operators in the DMAPI so this seemed useless if its not used.

Reply:
@AffectedArc07 
> We package `librustg.so` with the repo
> #### `librustg.so`
> ### `.so`

🤮